cypress/e2e/assets_spec/asset_homepage.cy.ts

```diff
@@ -90,7 +90,7 @@ describe("Asset Tab", () => {
     assetPage.selectImportOption();
     assetPage.selectImportFacility("Dummy Facility 1");
     assetPage.importAssetFile();
-    assetPage.selectImportLocation("Camera Locations");
+    assetPage.selectImportLocation("Camera Loc");
```

Comment:
the cypress failure is addressed in develop, and resolving the conflict, which will clear the issue